nemo_skills/inference/model/base.py

```
@@ -29,6 +29,309 @@
+            extra_body["repetition_penalty"] = params["repetition_penalty"]
+
+        # Add any additional extra_body parameters
+        if params["extra_body"]:
+            extra_body.update(params["extra_body"])
+
+        if extra_body:
+            request["extra_body"] = extra_body
+
+        return request
+
+    def build_completion_request_structure(self, prompt: str, params: dict) -> dict:
```

Ruff (0.13.1)

Unused method argument: `prompt`

(ARG002)